server/src/main/java/com/lunchtime/controllers/AuthController.java

```diff
@@ -0,0 +1,54 @@
+package com.lunchtime.controllers;
+
+
+import com.lunchtime.models.JwtAuthenticationToken;
+import com.lunchtime.models.JwtPersonDetails;
+import com.lunchtime.security.JwtUtil;
+import com.lunchtime.service.MyUserDetailsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+
+@RequestMapping
+public class AuthController {
+
+    @Autowired
+    private AuthenticationManager authenticationManager;
+
+    @Autowired
+    private JwtUtil jwtTokenUtil;
+
+    @Autowired
+    private MyUserDetailsService userDetailsService;
+
+
+    @PostMapping("/authenticate")
```

Comment:
Why don't you use prefix `/api`?

Reply:
allready fixed , i used this link because previously the server permissions were configured differently